blueprint/index.md

```diff
@@ -1,305 +1,345 @@
 
-![](images/028.png)
+![Confirmation of CSAT](images/029.png "Confirmation of CSAT")
 
-- ***Case 2*** - The Expert has expressed an opinion that they believe the conversation to have been completed and this event is detected via the Limitless update type = ‘cust\_confirmation’. If a CSAT hasn’t already been sent then this event triggers a CSAT link (shortened via a Data Action) to be sent to the Customer.
+- ***Case 3*** - A Limitless GigCX SmartCrowd status change event is detected via the Limitless GigCX update type = ‘state’. These status changes are further evaluated, and the following actions are performed:
+  - ***‘Escalated’*** - The expert crowd has decided they cannot resolve the question. When this state is detected, the customer receives a message stating an agent will be brought into the conversation, and a ‘Transfer to ACD’ block is used to bring in the agent.
+  - ***‘TimedOut’*** - The expert crowd did not respond in time. When this state is detected, the customer receives a message stating an agent will be brought into the conversation, and a ‘Transfer to ACD’ block is used to bring in the agent.
+  - ***‘Open’ & Not ‘TimedOut’*** - The Limitless GigCX lifecycle is in a state where a CSAT, if not already issued, should be sent to the customer. The flow actions this.
+  - ***‘Resolved’*** - The Limitless GigCX lifecycle was completed, and the question was resolved. This is an end state, and the flow disconnects.
 
-![](images/029.png)
+![Limitless SmartCrowd status change event](images/030.png "Limitless SmartCrowd status change event")
 
-- ***Case 3*** - A Limitless SmartCrowd status change event is detected via the Limitless update type = ‘state’. These status changes are further evaluated, and the following actions performed:
-  - ***‘Escalated’*** - The Expert Crowd have decided they cannot resolve this question. When this state is detected, the Customer receives a message stating that an Agent will be brought into the conversation and a ‘Transfer to ACD’ block is used to bring in the Agent.
-  - ***‘TimedOut’*** - The Expert Crowd did not respond in time. When this state is detected, the Customer receives a message stating that an Agent will be brought into the conversation and a ‘Transfer to ACD’ block is used to bring in the Agent.
-  - ***‘Open’ & Not ‘TimedOut’*** - The Limitless lifecycle is in a state where a CSAT, if not already issued, should be sent to the Customer. The flow actions this.
-  - ***‘Resolved’*** - The Limitless lifecycle has completed and the question was resolved. This is an end state and the flow disconnects.
+:::primary
+**Note**: The use of the Limitless GigCX flows has no impact on Cloud Analytics or reporting.
```

Comment:
Since you have it in a block you don't need the **Note**: in here.  Also I would get read of the line breaks before after the sentence